src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -62,6 +62,7 @@ See the LICENSE file in the project root for more information.
     <ItemGroup>
       <!-- Workaround for https://github.com/dotnet/corert/issues/7000 -->
       <IlcArg Include="--removefeature:CurlHandler" />
+      <IlcArg Include="--removefeature:Globalization" Condition="'$(StaticallyLinked)' == 'true'" />
```

Comment:
Could you please add a comment like `<!-- libicu loader in System.Globalization.Native is not compatible with static linking -->` in front of this line?

Reply:
added